frontend/justpostme/src/Routes.js

```diff
@@ -7,48 +7,47 @@ import WelcomePage from "./containers/welcomePage/WelcomePageContainer";
 import AboutPage from "./components/aboutPage/AboutPage";
 import SubmissionForm from "./containers/submissionForm/SubmissionFormContainer";
 import SubmissionInfo from "./containers/submissionInfo/SubmissionInfoContainer";
+import { Snackbar } from "./components/common/Snackbar";
 
 class Routes extends React.Component {
   render() {
-    const { loggedIn } = this.props;
+    const { loggedIn, showSnackbar, snackbarMessage } = this.props;
 
     return (
-      <Switch>
-        <Route
-          path="/login"
-          component={props => {
-            return loggedIn ? <ManagePages /> : <WelcomePage />;
-          }}
-        />
-        <Route path="/about" component={AboutPage} />
-        <Route
-          path="/pages"
-          render={props => {
-            return loggedIn ? (
-              <ManagePages />
-            ) : (
-              <Redirect
-                to={{ pathname: "/login", state: { from: props.location } }}
-              />
-            );
-          }}
-        />
-        <Route
-          path="/page/:id"
-          render={props => {
-            return loggedIn ? (
-              <PageControl />
-            ) : (
-              <Redirect
-                to={{ pathname: "/login", state: { from: props.location } }}
-              />
-            );
-          }}
-        />
-        <Route path="/form/:id" render={() => <SubmissionForm />} />
-        <Route path="/submission/:id" render={() => <SubmissionInfo />} />
-        <Redirect to="/login" />
-      </Switch>
+      <div>
+        <Snackbar showSnackbar={showSnackbar}>{snackbarMessage}</Snackbar>
+        <Switch>
+          <Route path="/login" component={WelcomePage} />
```

Comment:
conditional navigation went missing in your PR, look at the old state and copy and paste it as a render function, don't know what happend

Reply:
click on the code chunk to see the difference